### PR TITLE
Fonts, Nav, Home, & Footer styles

### PR DESCRIPTION
**Pull request by dunphyben**

### CODE REVIEW

File: src/features/Home/home.module.styl [hunk 1]
```diff
@@ -51,6 +54,10 @@
   $column(between)
   flex 1
   padding $padding-xl-horizontal $padding-xl-vertical $padding-xl-horizontal ($padding-xl-vertical * 3)
+  line-height $line-height-copy
+  color $text-description
+  font-weight 300
+  font-size 18px
```
Comment (BerkeleyTrue) [inline src/features/Home/home.module.styl]: css ordering

File: src/features/Home/home.module.styl [hunk 2]
```diff
@@ -70,6 +78,7 @@
 
 .prefooter
   $center(650px)
-  line-height 44px
+  line-height 34px
   padding 150px 0
-
+  font-weight 300
+  color $text-description
```
Comment (BerkeleyTrue) [inline src/features/Home/home.module.styl]: ordering

File: src/features/Home/home.module.styl (hunk 1 above)
Comment (dunphyben) [reply src/features/Home/home.module.styl]: Done!

File: src/features/Footer/footer.module.styl [hunk 3]
```diff
@@ -17,10 +17,15 @@
...
+    font-size 14px
+    font-weight 300
+    &:hover
+      color $ftr-hover
 
 .solutions
-  max-width 209px
+  max-width 260px
+  padding-top 22px
   h5
```
Comment (BerkeleyTrue) [inline src/features/Footer/footer.module.styl]: This header was changed from a h5 to an h4. Are the styles still relavent or can they be removed?

**Review by BerkeleyTrue — CHANGES_REQUESTED** [review]

**Pull request merged**